local-volume/provisioner/TODO.md

```diff
@@ -1,19 +1,16 @@
 # TODO
 
 ## P0
-* Update with Local volume API (msau42)
-* Give each provisioner a unique name (msau42)
 * Detect capacity of mount points (ddysher)
 * Deploy to public image repo (msau42)
 * E2E tests
 
 ## P1
+* Can we filter the PV watch to only certain fields?
```

Comment:
Ok thanks, from my initial search, it seems like the use of fieldSelector is very limited (not generic).  I will look into this as an enhancement for later.